src/sentry/lang/native/symbolicator.py

```diff
@@ -193,114 +191,6 @@ def process_payload(self, stacktraces, modules, signal=None):
         )
 
 
-def reverse_source_aliases(response, builtin_sources=None):
-    """Reverses internal source aliases from a response.
-
-    When an internal source was an alias this reverses the alias in the DIF candidates,
-    exposing it again like the user configured this.  This is only done for internal sources
-    since other sources do not support aliases.
-    """
-    if builtin_sources is None:
-        builtin_sources = settings.SENTRY_BUILTIN_SOURCES
-    reverse_aliases = reverse_aliases_map(builtin_sources)
-
-    for module in response.get("modules", []):
-        for candidate in module.get("candidates", []):
-            source_id = candidate["source"]
-            if source_id in reverse_aliases:
-                candidate["source"] = reverse_aliases.get(source_id)
-
-
-def reverse_aliases_map(builtin_sources):
```

Comment:
This has been moved verbatim to low lower in the file.

Reply:
What was the reason for moving this? It is now declared after use.

Reply:
I did this in an attempt to follow file's current style.  `get_options_for_project()` and `get_sources_for_project()` are used in `Symbolicator` and declared after it.  Hence these functions are used in `SymbolicatorSession` and declared after it.

Reply:
Thanks for explaining, makes sense 👍